app/views/events/index.html.haml

```diff
@@ -9,6 +9,12 @@
 .stripe
   .row
     .large-12.columns
-      - if @past_events.any?
+      - if @past_events_display.any?
         .past#past
-          = render partial: 'events', locals: { grouped_events: @past_events }
+          = render partial: 'events', locals: { grouped_events: @past_events_display }
+          Codebar has hosted 
+          = @all_past_events_count
+          weekly workshops, Codebar monthlies and all-day special events since 2013.
+          %br
```

Comment:
I've changed the scss file to add margin to #past instead - agreed that `<br>`s have no place here!